integration_tests/configs/yaml_doc.jsonnet

```diff
@@ -0,0 +1,9 @@
+// jsonnet yaml_doc.jsonnet -m . -S
```

Comment:
Is this file needed?

Reply:
no harms to keep, can use to generate equivalent yaml

Reply:
can we add a cli script to convert jsonnet to yaml, so we don't need to keep this?